CITATIONS.md

```diff
@@ -126,6 +126,34 @@
 
   > Sex.DetERRmine.py Lamnidis, T.C. et al., 2018. Ancient Fennoscandian genomes reveal origin and spread of Siberian ancestry in Europe. Nature communications, 9(1), p.5018. Available at: http://dx.doi.org/10.1038/s41467-018-07483-5. Download: https://github.com/TCLamnidis/Sex.DetERRmine
 
+- [MALT](https://www.nature.com/articles/s41559-017-0446-6)
+
+  > Vågene, Å.J., Herbig, A., Campana, M.G., Nelly, M., García, R., Warinner, C., Sabin, S., Spyrou, M.A., Valtueña, A.A., Huson, D., Tuross, N., Bos, K.I. & Krause, J. (2018). Salmonella enterica genomes from victims of a major sixteenth-century epidemic in Mexico. Nat Ecol Evol 2, 520–528. doi: [10.1038/s41559-017-0446-6](https://doi.org/10.1038/s41559-017-0446-6)
+
+- [HOPS](https://doi.org/10.1186/s13059-019-1903-0)
+
+  > Hübler, R., Key, F.M., Warinner, C. et al. (2019). HOPS: automated detection and authentication of pathogen DNA in archaeological remains. Genome Biol 20, 280. doi: [10.1186/s13059-019-1903-0](https://doi.org/10.1186/s13059-019-1903-0)
+
+- [MEGAN](https://doi.org/10.1101/gr.5969107)
+
+  > Daniel H. Huson, Alexander F. Auch, Ji Qi, and Stephan C. Schuster (2007). MEGAN analysis of metagenomic data. Genome Res. 17000, Published in Advance January 25, 2007, doi: [10.1101/gr.5969107](https://doi.org/10.1101/gr.5969107)
```

Comment:
This should be MEGAN6 citation: https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004957